openapi.yml

```
@@ -610,7 +619,7 @@ components:
             "href": "https://api.pa-onboarding.dev.io.italia.it/public-administrations/c_e044"
           }]
         }
-    Organization:
+    OrganizationToBeRegistered:
```

``
```suggestion
    Organization:
```

these are indeed _all_ the properties needed for the Organization object (imagine when you work with it in your code). 
`links` is a metadata of the API / HTTP protocol so the _other_ type (currently: `Organization`) should be named accordingly (ie. `OrganizationPayload` or `OrganizationResult`). see below